resources/js/Pages/Dashboard.vue

```diff
@@ -1,22 +1,122 @@
+                        <PrimaryButton class="col-md-4">
+                            Post
+                        </PrimaryButton>
+                    </form>
+                    <div class="post_items row g-3">
+                        <div class="post_item" v-for="(post, index) in posts">
+                            <div class="post_user">Charles</div>
+                            <span class="post_date">{{ post.created_at}}</span>
+                            <p class="post_content">{{ post.post_content}}</p>
+                            <span class="post_likes">
+                                <i class="zmdi zmdi-thumb-up"></i>Likes
+                            </span>
+                            <span class="post_comments_count" @click="showComments">
+                                <i class="zmdi zmdi-comment-text"></i>Comments
+                            </span>
+                            <div class="post_comments">
+                                <p class="post_comment">
+                                    <span class="d-block"><strong>ABC</strong></span>
+                                    <span class="d-block">Hello testing comment</span>                                
+                                </p>
+                                <p class="post_comment">
+                                    <span class="d-block"><strong>XYZ</strong></span>
+                                    <span class="d-block">Hello testing comment12</span>                                
+                                </p>
+                                <p class="post_comment">
+                                    <span class="d-block"><strong>PQR</strong></span>
+                                    <span class="d-block">Hello testing comment123</span>                                
+                                </p>
+                            </div>
+                            <div class="post_comment-box row g-3">
+                                <div class="col-md-8">
+                                    <TextInput
+                                        type="text"
+                                        placeholder="comment here"
+                                        v-model="comments[index]"
+                                    />
+                                </div>
+                                <PrimaryButton @click="postComment(comments[index], post.id)" class="col-md-4">
+                                    post comment
+                                </PrimaryButton>
+                            </div>
+                        </div><!-- post-item end -->
+                        <PrimaryButton class="offset-lg-5 col-md-3">
+                            Load More
+                        </PrimaryButton>
+                    </div>
                 </div>
             </div>
         </div>
     </AuthenticatedLayout>
 </template>
+<script>
+    const form = useForm({
+        post_content:''
+    });
```

Comment:
Are you trying to do some React stuff here?

Reply:
No, its an inertia stuff. Inertia allows to connect VUE/React with Laravel.
I replace above code as per Vue in recent commit.